app/controllers/tutor_comments_controller.rb

```diff
@@ -0,0 +1,52 @@
+class TutorCommentsController < ApplicationController
+  before_action :set_tutor_comment, only: [:edit, :update, :destroy]
+
+  def new
+    @tutor_comment = TutorComment.new
+    @tutor = Tutor.find(params[:tutor])
+  end
+
+  def create
+    @tutor_comment = TutorComment.new(tutor_comment_params)
+
+    if @tutor_comment.save
+      redirect_to tutor_path(tutor_comment_params[:tutor_id])
+    else
+      @tutor = Tutor.find(tutor_comment_params[:tutor_id])
```

Comment:
This branch (the `else` from line 14) is executed if the `save` fails. So in that case we need to re-render the `new` view and we need to find the tutor again so we can print the tutor's name at the top of the view.